Cleanup Helm Chart documentation (#62544)
mistercrunch/Airflow: chart/docs/manage-logs.rst

.. code-block:: bash

   helm upgrade --install airflow apache-airflow/airflow \
     --set logs.persistence.enabled=true
     # You can also override the other persistence
     # by setting the logs.persistence.* values
     # Please refer to values.yaml for details

Externally provisioned PVC
--------------------------

In this approach, Airflow will log to an existing ``ReadWriteMany`` PVC. You pass in the name of the volume claim to the chart
by using the ``logs.persistence.existingClaim`` parameter:

.. code-block:: bash

   helm upgrade --install airflow apache-airflow/airflow \
     --set logs.persistence.enabled=true \
     --set logs.persistence.existingClaim=my-volume-claim

.. note::

>    The volume have to be writable by the Airflow user. The easiest way is to ensure GID ``0`` has a write permission.

Reviewer: ```suggestion
   The volume has to be writable by the Airflow user. The easiest way to do this is to ensure GID ``0`` has write permission.
```
